test/utf8/OS_2977448.js

```diff
@@ -1,8 +1,14 @@
-//-------------------------------------------------------------------------------------------------------
-// Copyright (C) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
-//-------------------------------------------------------------------------------------------------------
-
-function TrimStackTracePath(line) {
-    return line && line.replace(/\(.+\\test.StackTrace./ig, "(");
-}
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+try {
```

Comment:
Looks like it got confused with the TrimStackTracePath file because it's also a small file and mostly similar thanks to the copyright header. This is a test migrated from full to core. It was in the Intl directory but it turned out not to be an Intl test but rather a Unicode test so I moved it to the utf8 directory (which also isn't a great fit since it's not about the encoding so much as the Unicode code point -- but it's a better fit anyway).

---
In reply to: [107275264](https://github.com/Microsoft/ChakraCore/pull/2726#discussion_r107275264) [](ancestors = 107275264)